www/source/javascripts/community.js

```diff
@@ -1,19 +1,128 @@
-var community_events = function(data) {
-    if ($(".community-events")) {
-        if (Array.isArray(data)) {
-            data.forEach(function(e) {
-                var ev = "<p>";
-                ev += e.title;
-                ev += "<br>";
-                ev += e.link;
-                ev += "<br>";
-                ev += e.pub_date;
-                ev += "<br>";
-                ev += e.description;
-                ev += "</p>";
-
-                $(".community-events").append(ev);
-            });
-        }
+$(function() {
+  var htmlDecode = function(input) {
```

Comment:
Ah, bummer. I wonder if that's something that can be fixed at the source. But if not, this seems fine -- although I might just `.replace("&#038;", "&")`, or build up the URL manually with the ID. That last one seems somehow safer to me, but yeah, it's a bit gross no matter which way you go, sigh. :)